assets/components/MainMenu.svelte

```diff
@@ -152,9 +154,21 @@ function onSearchKeydown(e) {
 function renderUnread(conversation, max = 60) {
   return conversation.unread > max ? (max + '+') : conversation.unread || 0;
 }
+
+function toggleSection(connection_id) {
+    if (collapsedStates[connection_id]) {
+        collapsedStates[connection_id] = !collapsedStates[connection_id];
+    } else {
+        collapsedStates[connection_id] = true; // default to collapsed if not already in the object
+    }
+
+    // Save to localStorage
+    localStorage.setItem('collapsedStates', JSON.stringify(collapsedStates));
+}
+
 </script>
 
-<style>
+<style lang="scss">
```

Comment:
Why is this changed?

Reply:
To be honest, I think it might've been from some earlier testing and hacking. I've removed that change for now, and will run through the test cases to see if I can replicate the reason why I put it in in the first place.